test/functional/p2p_ibd_txrelay.py

```diff
@@ -28,17 +28,17 @@
 )
 from test_framework.test_framework import BitcoinTestFramework
 
-MAX_FEE_FILTER = Decimal(9170997) / COIN
-NORMAL_FEE_FILTER = Decimal(100) / COIN
+MAX_FEE_FILTER = Decimal(9936506) / COIN
```

Comment:
seems pretty magical, what's going on with the feefilter change here?

Reply:
It's a magic number - the fee filter does some rounding and float nonsense to the input which is `MAX_MONEY`. I just put what made the test happy.

Reply:
Ah, the question is why the max filter changes when the min relay feerate changes. The fee filter rounder rounds the value and uses min relay feerate for spacing https://github.com/bitcoin/bitcoin/blob/9b1a7c3e8dd78d97fbf47c2d056d043b05969176/src/net_processing.cpp#L1889

We could assign a different value, but matching min / incremental relay feerate seems appropriate to me.